drivers/sensor/akm09918c/akm09918c.c

```diff
@@ -85,7 +105,7 @@ static int akm09918c_channel_get(const struct device *dev, enum sensor_channel c
 	} else if (chan == SENSOR_CHAN_MAGN_Z) {
 		akm09918c_convert(val, data->z_sample);
 	} else {
-		LOG_WRN("Invalid channel %d", chan);
+		LOG_DBG("Invalid channel %d", chan);
```

Comment:
Shouldn't this still be a LOG_WRN?

Reply:
I'm happy to revert this, it just got really spammy when running in the shell. We get back an error code, I didn't see a need for the extra print.